### PR TITLE
test: add test for schema registry protobuf

### LINKED ISSUE #10006: test: add a test for Protobuf + Confluent schema registry
You can add the test either in integration-tests or in e2e test.

### PR DESCRIPTION
**Pull request by waruto210**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Add a test for Protobuf + Confluent schema registry, close #10006

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: ci/scripts/run-e2e-test.sh [hunk 1]
```diff
@@ -81,6 +81,16 @@ RUST_BACKTRACE=1 target/debug/risingwave_e2e_extended_mode_test --host 127.0.0.1
 echo "--- Kill cluster"
 cargo make ci-kill
 
+echo "--- e2e, ci-1cn-1fe, protobuf schema registry"
+RUST_LOG="info,risingwave_stream=info,risingwave_batch=info,risingwave_storage=info" \
+cargo make ci-start ci-1cn-1fe &&
+python3 -m pip install psycopg2-binary, protobuf, confluent-kafka && \
+python3 e2e_test/schema_registry/pb.py && \
```
Comment (kwannoel) [inline ci/scripts/run-e2e-test.sh:88]:
```suggestion
cargo make ci-start ci-1cn-1fe
python3 -m pip install psycopg2-binary, protobuf, confluent-kafka
python3 e2e_test/schema_registry/pb.py
```

`set -e` at the start of the script will cause it to exit on error.

**Review by tabVersion — APPROVED** [review]
basically LGTM

File: e2e_test/schema_registry/pb.slt [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Before running this test:
+#   python3 e2e_test/schema_registry/pb.py
```
Comment (kwannoel) [inline e2e_test/schema_registry/pb.slt:2]:
```suggestion
# Before running this test, seed data into kafka:
#   python3 e2e_test/schema_registry/pb.py
```

**Review by kwannoel — APPROVED** [review]
LGTM, minor nits only.

**Review by fuyufjh — APPROVED** [review]
LGTM

File: ci/scripts/run-e2e-test.sh [hunk 3]
```diff
@@ -81,6 +81,16 @@ RUST_BACKTRACE=1 target/debug/risingwave_e2e_extended_mode_test --host 127.0.0.1
 echo "--- Kill cluster"
 cargo make ci-kill
 
+echo "--- e2e, ci-1cn-1fe, protobuf schema registry"
+RUST_LOG="info,risingwave_stream=info,risingwave_batch=info,risingwave_storage=info" \
+cargo make ci-start ci-1cn-1fe
+python3 -m pip install psycopg2-binary, protobuf, confluent-kafka
```
Comment (kwannoel) [inline ci/scripts/run-e2e-test.sh:87]:
```suggestion
python3 -m pip install psycopg2-binary protobuf confluent-kafka
```

**Pull request merged**